workbook/04-contributor.asciidoc

```diff
@@ -0,0 +1,178 @@
+== Video:  InnerSource Contributor
+
+=== SEGMENT: Becoming an InnerSource Contributor
+
+TIP:
+More than one answer may be correct in some questions.
+
+==== Question 1.  What is a benefit of becoming a contributor to an InnerSource project?
+
+. I can make sure others have to follow my suggestions to improve the project.
+. The host team will maintain the features I add on behalf of my team.
+. I can shape the solution to better fit my team’s needs.
+. I can help to shape all aspects of the project beyond the code itself (for example, GitHub branches and merges, tests, documentation).
+
+
+===== Correct answers 3 and 4
+
+Why 1 is incorrect: Open communities are not a dictatorship. As a contributor, you are part of a community helping the solution to grow and thrive.  This may involve compromise at times, but will pay longer term benefits to you and the community. 
+
+Why 2 is incorrect: In a shared solution you are invited to help improve the code, which includes bug fixes. Ownership in shared solutions should be seen as stewardship to keep the solution long term healthy.
```

Comment:
It could be read as that once you assume a "least-possible-effort" position, I agree. No-one ever saw those in the corporate space. ;)
Do you have an idea for a commit suggestion to bring this a little closer to the a "do protect the commons" position?